src/components/SortSelector/SortSelector.jsx

```diff
@@ -0,0 +1,60 @@
+import { useState, useRef, useEffect } from "react";
+import arrowDownIcon from "../../assets/icon/arrow-down-icon.svg";
+import sortIcon from "../../assets/icon/sort-icon.svg";
+import styles from "./SortSelector.module.css";
+
+const SortSelector = ({ onChange }) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [selected, setSelected] = useState("최신순");
+  const dropdownRef = useRef(null);
+
+  const sortMap = {
+    최신순: "recent",
+    좋아요순: "favorite",
+  };
+
+  const handleSelect = (label) => {
+    setSelected(label);
+    setIsOpen(false);
+    onChange(sortMap[label]);
+  };
+
+  useEffect(() => {
+    const handleClick = (e) => {
+      if (dropdownRef.current && !dropdownRef.current.contains(e.target)) {
+        setIsOpen(false);
+      }
+    };
+    document.addEventListener("click", handleClick);
+    return () => document.removeEventListener("click", handleClick);
+  }, []);
+
+  return (
+    <div ref={dropdownRef} className={styles.sortWrapper}>
+      <button
+        onClick={() => setIsOpen((prev) => !prev)}
+        className={styles.dropdownButton}
+      >
+        <span className={styles.selectedText}>{selected}</span>
+        <img src={arrowDownIcon} alt="다운 버튼" className={styles.downIcon} />
+        <img src={sortIcon} alt="모바일 정렬" className={styles.sortIcon} />
+      </button>
+
+      {isOpen && (
+        <ul className={styles.dropdownList}>
+          <li onClick={() => handleSelect("최신순")} className={styles.recent}>
+            최신순
+          </li>
```

Comment:
❗️ 수정요청
이런 클릭 가능한 요소의 경우 버튼을 사용해주세요. 더 적절한 태그가 있는데 이미지 태그에 onClick 을 주게되면 button에게 기대하는 동작을 제대로 수행하지 못합니다!

```suggestion
          <li className={styles.recent}>
            <button type="button" value={sortMap["최신순"]} onClick={handleSelect}>최신순</button>
          </li>
```

위에처럼 button 태그로 바꾸시고 onClick 시 event 객체에서 button 태그의 value를 받아오는 방식으로 구현하시는 것을 추천드려요!